src/components/custom/signup.js

```diff
@@ -146,6 +146,7 @@ class Signup extends Component {
 
             binary_socket.close()
         }
+        navigate('singup-success')
```

Comment:
the page route should be `signup-success`

Reply:
We cannot set this navigate to this component as it affects the `/signup` page as well. We need to do this in the signup-public.js